static/src/javascripts/lib/ajax.js

```diff
@@ -13,7 +12,7 @@ const ajax = (params) => {
         options.crossOrigin = true;
     }
 
-    const r = reqwest(options);
+    const r = fetch(options);
```

Comment:
I'm wondering if the API of `fetch` is the same as `reqwest` for clients of `ajax.js` ?
For example are all the options passed to `ajax` supported by fetch in here:
https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/identity/api.js